exporter/kafkaexporter/raw_marshaler.go

```diff
@@ -0,0 +1,101 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kafkaexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/kafkaexporter"
+
+import (
+	"errors"
+
+	"github.com/Shopify/sarama"
+	jsoniter "github.com/json-iterator/go"
```

Comment:
While I prefer this over the default encoding package, there is subtle differences between them

If we are going to use this, we have to use throughout the exporter.

Reply:
It is used elsewhere, such as for HEC exporter. Happy to scrub it out if it helps land the PR, and maybe later open it back up with a performance benchmark.